### PR TITLE
updated typings

### PR DESCRIPTION
**Pull request by skysteve**
Please fill in this template.

- [x] Prefer to make your PR against the `types-2.0` branch.
- [x] The package does not provide its own types, and you can not add them.
- [x] Test the change in your own code.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes).

If adding a new definition:
- [ ] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [ ] Run `tsc` without errors.
- [ ] Include the required [files](https://github.com/DefinitelyTyped/DefinitelyTyped#create-a-new-package) and header.

If changing an existing definition:
- [x] Provide a URL to  documentation or source code which provides context for the suggested changes: <<http://pieroxy.net/blog/pages/lz-string/index.html>>
- [x] Increase the version number in the header if appropriate.

Adding missing typings as documented http://pieroxy.net/blog/pages/lz-string/index.html

### CODE REVIEW

File: lz-string/index.d.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-﻿// Type definitions for lz-string v1.3.3
+﻿// Type definitions for lz-string v1.3.4
```
Comment (M0ns1gn0r) [inline lz-string/index.d.ts]: The compressToEncodedURIComponent and decompressFromEncodedURIComponent methods appeared in lz-string 1.3.5.

File: lz-string/index.d.ts [hunk 2]
```diff
@@ -57,5 +57,35 @@ declare namespace LZString {
...
+
+        /**
+        * produces an uint8Array
+        *
+        * @param uncompressed A string which should be compressed.
+        */
+        compressToUint8Array(uncompressed: string): Uint8Array;
+
+        /**
+        * Decompresses "valid" input string created by the method compressToUint8Array().
```
Comment (M0ns1gn0r) [inline lz-string/index.d.ts]: Should be: Decompresses "valid" array created by the method compressToUint8Array().

**Review by M0ns1gn0r — CHANGES_REQUESTED** [review]
A few small things need to be changed.

**Comment by skysteve** [comment]
Fixed :)

**Comment by M0ns1gn0r** [comment]
👍 LGTM

**Pull request merged**